### PR TITLE
User can choose language for site

### PR DESCRIPTION
**Pull request by kaylawoodbury**
# [PT Story](https://www.pivotaltracker.com/story/show/171811128)
## User Story
```
As a user
In order to view the application in my preferred language
I would like to be able choose between English and Swedish
```
## Changes proposed in this pull request:
* Adds file for i18next configuration
* Adds locale files for english and swedish
* Updates all site navigation text to use the i18n translation
## What I have learned working on this feature:
* How to implement i18n
## Packages/Gems added
- i18next
- i18next-browser-languagedetector
- i18next-xhr-backend
- react-i18next
## Screenshots (Only if client side story)
Screenshot 2020-04-02 at 11 01 02
Screenshot 2020-04-02 at 11 01 11

## Lo-Fi (Only if client side story)
Pasted Image at 2020_03_17_05_51 am

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
   <meta name="theme-color" content="#000000" />
   <meta name="description" content="Web site created using create-react-app" />
   <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
-  <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+  <!-- <link rel="manifest" href="%PUBLIC_URL%/manifest.json" /> -->
```
Comment (robin-lillqvist) [inline public/index.html]: Do we need to keep this commented out file?

**Review by robin-lillqvist — COMMENTED** [review]
commented

**Review by robin-lillqvist — APPROVED** [review]
APPROVED

**Review by blake-futchi — APPROVED** [review]
Looks great!

**Pull request merged**